docs/gen.log

```diff
@@ -0,0 +1,1062 @@
+[
```

Comment:
This should be ignored. You should modify `.gitignore` to have `gen.log` rather than `dev.log`